app/Http/Requests/Nordigen/ConnectNordigenBankIntegrationRequest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2023. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+namespace App\Http\Requests\Nordigen;
+
+use App\Http\Requests\Request;
+use App\Libraries\MultiDB;
+use App\Models\Company;
+use App\Models\User;
+use Cache;
+use Log;
+
+class ConnectNordigenBankIntegrationRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'lang' => 'string',
+            'institution_id' => 'string',
+            'redirect' => 'string', // TODO: @turbo124 @todo validate, that this is a url without / at the end
```

Comment:
If this is not user input (which for one_time_token it never is) then you can ignore validation here as it has been prebuilt in the PrepareForValidation method

Reply:
In production this would be protected by the api token, and also these fields would be passed in via the nordigen response, so I don't see much value here to use validation.

Reply:
I guess we could remove the value completly, but I had placed it there to be used, when the frontend needs a different redirect. Like I said f.ex. for refreshing an account and redirect to the same account afterwards or be able to check if it was successful for this account.
Or do you mean just to validate if its an url? If its wrong it wouldnt matter?